src/components/Common/PaginationControl/PaginationControl.tsx

```diff
@@ -39,18 +38,20 @@ export const PaginationControl = ({
       <Flex justifyContent="space-between" alignItems="center">
         <div className={style.paginationControlCount}>
           <section>
-            <Select
-              control={control}
-              aria-label={t('labels.paginationControllCountLabel')}
+            <SelectField
               name="pageSize"
-              onSelectionChange={n => {
+              label={t('labels.paginationControllCountLabel')}
+              aria-label={t('labels.paginationControllCountLabel')}
```

Comment:
I think this should be updated as follows yeah?

```suggestion
              shouldVisuallyHideLabel
```

Not sure why it isn't throwing a type error for `aria-label`